docs: Update readme for mainnet (#3132)
chainwayxyz/citrea: README.md

# Citrea

**The first rollup that enhances the capabilities of Bitcoin blockspace with zero-knowledge technology, now [live on Bitcoin Mainnet](https://www.blog.citrea.xyz/citrea-mainnet-is-live/)! 🎉🍊🍋**

![](resources/assets/banner.png)

> [!WARNING]
> Follow our [website](https://citrea.xyz) & [social media accounts](https://twitter.com/citrea_xyz) for announcements regarding the next phases of Citrea. \
> \
> Citrea uses **BTC** as its native token. **There's no Citrea token**. Please beware of scams!

> 

Reviewer: swithc these two lines, second one is the actual warning now

To see all past audits, visit: [Audits & Security](https://docs.citrea.xyz/security/audits-inquiries)

## Bug Bounty
Citrea maintains active bug bounty programs. Before diving into the codebase, researchers should review the [Auditor's Guide](docs/auditors-guide.md), then head to the [Citrea Active Bug Bounties](https://docs.citrea.xyz/security/audits-inquiries#active-bug-bounties) to see the currently available bug bounties.

## FAQ

| Question                                         | Answer                                                                                                                      |
| ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------- |
| How do I set up the development environment?     | [dev-setup.md](./docs/dev-setup.md)                                                                                         |
| How do I run Citrea for testing and development? | [run-dev.md](./docs/run-dev.md)                                                                                                     |

> | Where can I read more about the architecture?    | Go to [SUMMARY.md](./docs/SUMMARY.md) and navigate from there to the subject you want to read about, also refer to [our official documentation.](https://docs.citrea.xyz) for the non-technical side |

Reviewer: ```suggestion
| Where can I read more about the architecture?    | Go to [SUMMARY.md](./docs/SUMMARY.md) and navigate from there to the subject you want to read about, or refer to [our official documentation](https://docs.citrea.xyz). |
```
